### PR TITLE
--output_format Arg added for JSON/CLI output

### PR DESCRIPTION
**Pull request by am2367**
--output_format argument added for JSON/CLI output and storage. Storage function created for storing the JSON results. Readme examples updated.

### CODE REVIEW

**Comment by tradytics** [comment]
Excellent, merging it now. I hope you tested it? :P <3

File: detection_engine.py [hunk 1]
```diff
@@ -52,10 +53,11 @@
 is_test = args.is_test
 future_bars = args.future_bars
 volatility_filter = args.volatility_filter
+output_format = args.output_format
```
Comment (Und3rf10w) [inline detection_engine.py:56]: I would recommend handling cases where the user passes in the argument with different casing, e.g. `JSON` vs `json`. The most simple way (read: Least amount of code changes) would be to `output_format = args.output_format.upper()`

**Comment by am2367** [comment]
Yes, it has been tested using the commands you have included in the ReadMe.

File: detection_engine.py (hunk 1 above)
Comment (am2367) [reply detection_engine.py:56]: Fair point, I can make that update.

Comment (am2367) [reply detection_engine.py]: Updated

**Comment by tradytics** [comment]
Perfect, thank you. Merging now.

**Pull request merged**